packages/fungible-token/bridge-fungible-token/tests/functions/message_receiver/mod.rs

```diff
@@ -48,20 +50,23 @@ mod success {
                 .with_BRIDGED_TOKEN_GATEWAY(Bits256::from_hex_str(message_sender).unwrap())
                 .unwrap();
 
+        let (proxy_id, _implementation_contract_id) =
+            get_contract_ids(&wallet, Some(configurables.clone()));
+
         let (message, coin, deposit_contract) = create_deposit_message(
             token_address,
             token_id,
             from_address,
             *recipient,
             U256::from(amount),
             BRIDGED_TOKEN_DECIMALS,
-            Some(configurables.clone()),
+            proxy_id,
             false,
             None,
         )
         .await;
 
-        let (bridge, utxo_inputs) = setup_environment(
+        let (implementation_contractid, bridge, utxo_inputs) = setup_environment(
```

Comment:
```suggestion
        let (implementation_contract_id, bridge, utxo_inputs) = setup_environment(
```
only a nit, feel free to ignore if it would delay